app/pages/lab/workflow.cjsx

```diff
@@ -70,6 +70,8 @@ EditWorkflowPage = React.createClass
 
     noTasksDefined = Object.keys(@props.workflow.tasks).length is 0
 
+    stats_completeness_type = @props.workflow.configuration.stats_completeness_type ? 'retirement'
```

Comment:
As far as I can tell react does not let you define defaults for nested props.  As long as `workflow` is passed in as a prop anything assigned to `workflow` in `getDefaultProps` is ignored.